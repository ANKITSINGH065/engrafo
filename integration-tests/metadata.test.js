var utils = require("./utils");

test("footnotes after metadata are inlined inside metadata", done => {
  utils.expectBodyToMatchSnapshot("metadata/footnotes-after-metadata.tex", done);
});
