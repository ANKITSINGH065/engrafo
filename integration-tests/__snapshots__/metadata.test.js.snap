// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footnotes after metadata are inlined inside metadata 1`] = `
<body>
  <div
    class="engrafo-container"
  >
    

    <dt-article>
      

      <h1>
        Hello
      </h1>
      <div
        class="engrafo-metadata"
      >
        <div
          class="authors"
        >
          <div
            class="author"
          >
            <span>
              Joe Bloggs
            </span>
          </div>
          <div
            class="author"
          >
            <span>
              Jane Smith
            </span>
          </div>
          <p>
            Some University
          </p>
        </div>
        <div
          class="engrafo-metadata-custom"
        />
      </div>
      





    </dt-article>
    

    <dt-appendix>
      

      


      <div
        class="l-body"
      />
      

    </dt-appendix>
    



    <div
      id="footnote-hover-boxes-container"
    />
  </div>
</body>
`;
